response_operations_ui/templates/admin/user-create.html

```diff
@@ -0,0 +1,311 @@
+
+{% extends "layouts/base.html" %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/lists/_macro.njk" import onsList %}
+{% from "components/password/_macro.njk" import onsPassword %}
+{% from "components/input/_macro.njk" import onsInput %}
+{% from "components/fieldset/_macro.njk" import onsFieldset %}
+{% from "components/table/_macro.njk" import onsTable %}
+{% from "components/checkboxes/_checkbox-macro.njk" import onsCheckbox %}
+
+{% set page_title = 'Create New Account' %}
+
```

Comment:
![Screenshot 2022-07-11 at 13 17 43](https://user-images.githubusercontent.com/14179817/178262389-19974fac-1bf0-4b0b-956c-0e091369b02f.png)

Seems like there is a bigger gap then normal at the top and it's a bit squashed at the bottom

Reply:
All fixed

Reply:
Padding at the bottom looks fine 👍 , but mine still seems to have a bigger gap at the top then norma
![Screenshot 2022-07-14 at 09 09 38](https://user-images.githubusercontent.com/14179817/178934540-e13cfe98-1995-4bef-a6fa-44f2e2f3ff96.png)
l